packages/node-resolve/src/index.js

```diff
@@ -29,6 +29,10 @@ const deepFreeze = (object) => {
 
   return object;
 };
+
+const baseConditions = ['default', 'module']
```

Comment:
@lukastaegert should we match `module` also in the case of require? Or only when using import?

Reply:
Good question. I know @sokra has written something up how they handle the module condition in webpack but I cannot seem to find the document 🙄. But to my understanding, the main point of the `module` condition is to denote that it is "safe" to use this ES module to resolve a `require` statement, so yes, I'd say the condition should be added.